pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/download/SampleQueueTest.java

```diff
@@ -0,0 +1,178 @@
+package org.broadinstitute.dsm.download;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import java.util.Map;
+
+import com.google.gson.Gson;
+import lombok.extern.slf4j.Slf4j;
+import org.broadinstitute.dsm.DbAndElasticBaseTest;
+import org.broadinstitute.dsm.db.dao.ddp.instance.DDPInstanceDao;
+import org.broadinstitute.dsm.db.dto.ddp.instance.DDPInstanceDto;
+import org.broadinstitute.dsm.db.dto.ddp.participant.ParticipantDto;
+import org.broadinstitute.dsm.model.filter.participant.ManualFilterParticipantList;
+import org.broadinstitute.dsm.model.participant.ParticipantWrapperDto;
+import org.broadinstitute.dsm.service.admin.UserAdminTestUtil;
+import org.broadinstitute.dsm.service.download.DownloadParticipantListService;
+import org.broadinstitute.dsm.util.ElasticTestUtil;
+import org.broadinstitute.dsm.util.TestParticipantUtil;
+import org.broadinstitute.dsm.util.TestUtil;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import spark.QueryParamsMap;
+
+@Slf4j
+public class SampleQueueTest extends DbAndElasticBaseTest {
+
+    private static final String instanceName = "download_test_instance";
+    private static final String groupName = "download_test_group";
+    static String userEmail = "downloadUser@unittest.dev";
+    static int userId;
+    private static String esIndex;
+    private static DDPInstanceDto ddpInstanceDto;
+    private static String guid = "PT_SAMPLE_QUEUE_TEST";
+    private static ParticipantDto participantDto = null;
+    private DownloadParticipantListService downloadParticipantListService = new DownloadParticipantListService();
+    static UserAdminTestUtil userAdminTestUtil = new UserAdminTestUtil();
+
+    @BeforeClass
+    public static void doFirst() {
+        esIndex = ElasticTestUtil.createIndex(instanceName, "elastic/lmsMappings.json", null);
+        userAdminTestUtil.createRealmAndStudyGroup(instanceName, instanceName, "downloadTest", groupName, esIndex);
```

Comment:
I don't think you need `UserAdminTestUtil`. Something like this should be sufficient:
`        ddpInstanceDto = DdpInstanceGroupTestUtil.createTestDdpInstance(instanceName, esIndex);`

And then in the teardown:
`        DdpInstanceGroupTestUtil.deleteInstance(ddpInstanceDto);`